### PR TITLE
Demo: Add sample code for Bootstrap 4 usage

### PR DESCRIPTION
**Pull request by gwhenne**
Ref #2131.

Simple page showing how to use jquery-validation with Bootstrap 4.

### CODE REVIEW

File: demo/bootstrap/index-bs4.html [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+					email: "Please enter a valid email address",
+					agree: "Please accept our policy"
+				},
+				errorElement: "em",
+				errorPlacement: function ( error, element ) {
+					// Add the `help-block` class to the error element
+					error.addClass( "invalid-feedback" );
+
+					if ( element.prop( "type" ) === "checkbox" ) {
+						error.insertAfter( element.parent( "label" ) );
```
Comment (Arkni) [inline demo/bootstrap/index-bs4.html]: `element.parent( "label" )` => `element.next( "label" )`

File: demo/bootstrap/index-bs4.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+					} else {
+						error.insertAfter( element );
+					}
+				},
+				highlight: function ( element, errorClass, validClass ) {
+					$( element ).parents( ".col-sm-5" ).addClass( "is-invalid" ).removeClass( "is-valid" );
```
Comment (Arkni) [inline demo/bootstrap/index-bs4.html]: Remove `.parents( ".col-sm-5" )`, it's not needed here.

File: demo/bootstrap/index-bs4.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+				},
+				unhighlight: function (element, errorClass, validClass) {
+					$( element ).parents( ".col-sm-5" ).addClass( "is-valid" ).removeClass( "is-invalid" );
```
Comment (Arkni) [inline demo/bootstrap/index-bs4.html]: Remove `.parents( ".col-sm-5" )`, it's not needed here.

File: demo/bootstrap/index-bs4.html [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+									<input type="password" class="form-control" id="confirm_password" name="confirm_password" placeholder="Confirm password" />
+								</div>
+							</div>
+
+							<div class="form-group row">
+								<div class="col-sm-6 offset-sm-4">
+									<div class="form-check">
+										<label>
+											<input type="checkbox" id="agree" name="agree" value="agree" class="form-check-input"/>Please agree to our policy
+										</label>
```
Comment (Arkni) [inline demo/bootstrap/index-bs4.html]: Change this to:
```html
<input type="checkbox" id="agree" name="agree" value="agree" class="form-check-input"/>
<label class="form-check-label">Please agree to our policy</label>
```

**Comment by gwhenne** [comment]
@Arkni - I just uploaded code with your suggestions - thank you!

I didn't add Bootstrap 3 style feedback icons to the sample. Bs3 used Glyph icons to do this, which are not in Bs4. 

From https://www.quackit.com/bootstrap/bootstrap_4/differences_between_bootstrap_3_and_bootstrap_4.cfm
> Validation styles are not available for Bootstrap 4 forms. Use custom Bootstrap form validation messages instead.

**Comment by Arkni** [comment]
I applied the changes you pushed (found in branch `patch-2` in your fork) [manually and pushed them to the branch `patch-1`](https://help.github.com/articles/committing-changes-to-a-pull-request-branch-created-from-a-fork/), because when you update the file using GitHub interface, it creates a new branch for each update, that's why I didn't see the changes here.

Thanks a lot for your contribution :)

**Pull request merged**